nixos/modules/services/desktops/deepin/deepin-menu.nix

```diff
@@ -0,0 +1,31 @@
+# deepin-menu
+
+{ config, pkgs, lib, ... }:
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.deepin.deepin-menu = {
+
+      enable = lib.mkEnableOption
+        "Dbus service for the unified menu service for Deepin Desktop Environment";
+
+    };
+
+  };
+
+
+  ###### implementation
+
+  config = lib.mkIf config.services.deepin.deepin-menu.enable {
+
+    environment.systemPackages = [ pkgs.deepin.deepin-menu ];
```

Comment:
Do programs in deepin expect `deepin-menu` in `PATH`?

If not we can remove this.

Reply:
I am not sure. `deepin-menu` installs `$out/bin/deepin-menu`, which is used by the dbus service.

Reply:
The exec value of the service is hardcoded, so I think it should be fine.

Reply:
From the already packaged projects that uses it (`deepin-terminal` and `deepin-wm`) none seems to reference the binary directly.

Therefore I am going to remove it from `systemPackages`.

Reply:
Done.